packages/vote/opinion/opinion.go

```diff
@@ -94,3 +94,14 @@ func ConvertOpinionsToInts32(opinions []Opinion) []int32 {
 	}
 	return result
 }
+
+// ConvertOpinionToFloat64 converts the given Opinion to an float64.
+func ConvertOpinionToFloat64(x Opinion) float64 {
+	switch {
+	case x == Like:
+		return 1
+	case x == Dislike:
+		return 2
+	}
+	return 4
```

Comment:
why 4?

Reply:
this value was used in ConvertOpinionToInt32, not sure why maybe better to use -1? 
And add check: opinion>0 in biasTowardsOwnOpinion()
Anyway, I see I also need to change Like to 1 and Dislike to 0 (according to FPC paper)

Reply:
it would be more consistent. but check with Angelo why there is this current numbering, maybe there is a reason for this